skyrim-platform/src/tilted/ui/DX11RenderHandler.cpp

```diff
@@ -1,21 +1,21 @@
+#include "TextToDraw.h"
 #include <DX11RenderHandler.h>
-#include <OverlayClient.h>
-
-#include "../ui/TextToDraw.h"
 #include <DirectXColors.h>
 #include <DirectXTK/CommonStates.h>
 #include <DirectXTK/DDSTextureLoader.h>
 #include <DirectXTK/SimpleMath.h>
 #include <DirectXTK/SpriteBatch.h>
 #include <DirectXTK/SpriteFont.h>
 #include <DirectXTK/WICTextureLoader.h>
+#include <OverlayClient.h>
 #include <cmrc/cmrc.hpp>
-#include <string>
-
 #include <functional>
 #include <iostream>
+#include <string>
 
 CMRC_DECLARE(skyrim_plugin_resources);
+using ObtainTextsToDrawFunction = std::function<void(
```

Comment:
Please do not declare ObtainTextsToDrawFunction twice in the project. Declare somewhere once. in TextToDraw.h for example